docs/src/cfs_versions.dox

```diff
@@ -4,93 +4,64 @@
+  Note the Mission Version number is provided for missions to be able to identify unique changes they
+  have made to the released software (via clone and own).
 
   <H2> Identifying Development Builds </H2>
 
   In order to distinguish between development versions, we also provide a BUILD_NUMBER.
 
   The BUILD_NUMBER reflects the number of commits since the BUILD_BASELINE, a baseline git tag, for each particular
-  component. The BUILD_NUMBER integer increases monotonically for a given development cycle. The BUILD_BASELINE identifies the current development cycle and is a git tag with format vX.Y.Z. The Codename used in the version string also refers to the current development cycle. When a new baseline tag and codename are created, the the BUILD_NUMBER resets to zero and begins increasing
-  from a new baseline.
-
-  <H2> Templates for the version and version string </H2>
-
-  The following templates are the code to be used in cfe_version.h for either official releases or development builds. The apps and repositories follow the same pattern by replacing the CFE_ prefix with the appropriate name; for example, osal uses OS_, psp uses CFE_PSP_IMPL, and so on.
-
-  <H3> Template for Official Releases </H3>
-
-  \verbatim
-
-  /*<! Official Release Version Number */
-  #define CFE_SRC_VERSION \
-      CFE_STR(CFE_MAJOR_VERSION) "." \
-      CFE_STR(CFE_MINOR_VERSION) "." \
-      CFE_STR(CFE_REVISION) "."      \
-      CFE_STR(CFE_MISSION_REV)
-
-  #define CFE_VERSION_STRING \
-      "cFE version " CFE_SRC_VERSION
-
-  \endverbatim
-
-  <H3> Template for Development Builds </H3>
-
-  \verbatim
-
-  /*! @brief Development Build Version Number.
-   * Baseline git tag + Number of commits since baseline. @n
-   * See cfs_versions.dox for format differences between development and release versions.
-   */
-  #define CFE_SRC_VERSION \
-      CFE_BUILD_BASELINE CFE_STR(CFE_BUILD_NUMBER)
-
-  /*! @brief Development Build Version String.
-   * Reports the current development build's baseline, number, and name. Also includes a note about the latest official version. @n
-   * See cfs_versions.dox for format differences between development and release versions.
-  */
-  #define CFE_VERSION_STRING                                                          \
-      " cFE Development Build "                                                     \
-      CFE_SRC_VERSION " (Codename: CONSTELLATION_NAME)"       /* Codename for current development */  \
-      ", Last Official Release: cfe vX.Y.Z"   /* For full support please use this version */
-
-  \endverbatim
+  component. The BUILD_NUMBER integer increases monotonically for a given development cycle. The BUILD_BASELINE
```

Comment:
consider replacing with 

"monotonically increases for a given baseline"

Reply:
Fixed in 5394484.